src/NuGet.Core/NuGet.Configuration/NuGet.Configuration.csproj

```diff
@@ -7,6 +7,7 @@
     <NoWarn>$(NoWarn);CS1591</NoWarn>
     <TargetFrameworks>$(TargetFrameworksLibrary)</TargetFrameworks>
     <TargetFrameworks Condition="'$(IsBuildOnlyXPLATProjects)' != 'true'">$(TargetFrameworks);net45</TargetFrameworks>
+    <TargetFrameworks Condition="'$(IsBuildOnlyXPLATProjects)' == 'true'">$(TargetFrameworks);netstandard2.0</TargetFrameworks>
```

Comment:
if my suggestion in common.project.props can't be taken, given that the last two of these are opposites of each other, can we get rid of the first, non-conditioned, declaration, so MSBuild doesn't have to overwrite or re-evaulate the same property multiple times in all scenarios?  I'm not enough of an MSBuild expert to know whether or not it has an actual performance benefit, but maybe it's clearer to readers to me because I will need to do less mental arithmetic to figure out which set declarations contribute to the final value.